app/client/templates/single-app/single-app.scss

```diff
@@ -42,14 +42,12 @@ $app-container-padding-rl: 4.6rem;
         position: relative;
         height: 85%;
         width: 23rem;
-        min-width: 15rem;
 
         .image {
 
           content: "";
           background-size: contain;
           background-repeat: no-repeat;
-          background-position: center;
```

Comment:
I'm a little worried that removing the `center` here will change how the image appear. Did you test what happens when you click an app's mini screenshot? So long as that still looks reasonable when you click on the image, great, but do test it.